test/functional/test_framework/test_node.py

```diff
@@ -91,6 +91,12 @@ def wait_for_rpc_connection(self):
             attempts -= 1
         raise AssertionError("Unable to connect to bitcoind")
 
+    def get_wallet_rpc(self, wallet_name):
+        assert self.rpc_connected
+        assert self.rpc
+        wallet_path = "wallet/%s" % wallet_name
+        return self.rpc / wallet_path
```

Comment:
What does this do? `wallet_path` is for sure a string, but how do you divide the rpc object by a string?